cmds/wget/wget.go

```diff
@@ -5,37 +5,73 @@
 // Wget reads one file from a url and writes to stdout.
 //
 // Synopsis:
-//     wget URL
+//     wget [ARGS] URL
+//
+// Description:
+//     Returns a non-zero code on failure.
+//
+// Options:
+//     -O: output filename, defaults to '-' (stdout)
```

Comment:
I'd prefer not implementing a `-O` at all and just relying on the shell's redirect `wget http://example.com > out.html`.

I only implemented it so I could do a parity test with GNU's wget: `EXECPATH=wget go test`.

Thinking about it more, a better solution is to get rid of `-O` altogether, and make it so the parity test can be run like:
`EXECPATH="wget -O -" go test`.